test/test_run_analysis.rb

```diff
@@ -15,14 +15,13 @@ def before_setup
     buildstock_directory = File.join(File.dirname(__FILE__), '..')
 
     @testing_baseline = File.join(buildstock_directory, 'testing_baseline')
-    @national_baseline = File.join(buildstock_directory, 'national_baseline')
     @testing_upgrades = File.join(buildstock_directory, 'testing_upgrades')
```

Comment:
I was going to suggest deleting this too, but appears its used in some tests below. Suggest a name change so that it's not confused with the "testing project"? Maybe that's overkill; your call.

Reply:
I moved the `@testing_upgrades` variable down to the test where the variable is called since it is the only time the variable is used. I also renamed the output folder in the yaml file. [082c1b5](https://github.com/NREL/resstock/pull/1329/commits/082c1b55020524f4ef1a7f67631453ffc0b2a879)